clients/java/dkv-client/src/test/resources/dkv_config.yaml

```diff
@@ -0,0 +1,25 @@
+access-logs : ""
+database : "default"
+db-engine : "rocksdb"
+db-engine-ini : ""
+block-cache-size : 3221225472
+root-folder : "/tmp/dkvsrv"
+dc-id : "default"
+disable-auto-master-disc : false
+discovery-service-config : "internal/discovery/discovery.ini"
+diskless : false
+pprof : false
+repl-master-addr : ""
+repl-poll-interval : "2s"
+role : ""
+statsd-addr : ""
+vbucket : "default"
+verbose : false
+nexus-cluster-name : "default"
+nexus-cluster-url : ""
+nexus-max-snapshots : 5
+nexus-max-wals : 5
+nexus-repl-timeout : 5
+nexus-lease-based-reads : true
+nexus-snapshot-count : 100
+nexus-snapshot-catchup-entries : 5
```

Comment:
Given this is YAML format, it can also be written as 

```yaml
nexus:
    cluster-name : xxx
```

This way nexus config could also be abstracted out.

Reply:
Sure. If nested structures are supported then it's alright.